### DISCUSSION TITLE
"Random subseed" button missing?

### DISCUSSION THREAD

**Discussion by noisefloordev** [vladmandic/sdnext#209] [General]
The random_subseed button is missing.  I use it all the time: generate variations until I find one that's useful, lock in the subseed so I can render it with hires fix on, then set it back to random so I can continue generating variations of the primary seed.  (I then comp a bunch of variations together to create a starting image for inpainting and tweaking.)  Without it I have to manually edit the subseed back to -1 every time, which is clunky.

**Comment by vladmandic** [comment]
it was very very rarely used so i removed it. i know it seems tiny so why remove, but i'm trying to remove everything that is non-essential to clean up ui a bit, it was getting very busy.
but you can change the ui theme back to gradio/default if that helps?

**Comment by noisefloordev** [comment]
I have it on default, but that's just the visual theme.  I do use that button often.  I can put it back in manually, it was just surprising that it was missing.

Maybe the "load seed" button should set the subseed to random, and the "load subseed" button should load both (no reason to load a subseed without a matching seed).  That'd allow resetting the subseed to random without needing a 4th button.

(There are a ton of things that could be cleaned up in the UI, it'd be great if the inpaint page didn't require scrolling through 3 pages to access anything...)

**Reply by vladmandic** [reply]
> I have it on default, but that's just the visual theme

You can always unhide in your `user.css`, that applies on top of the theme.

> Maybe the "load seed" button should set the subseed to random

Hmm...I'll think about that...

> There are a ton of things that could be cleaned up in the UI 

Oh, yes, this is just a tip of the iceberg

**Comment by noisefloordev** [comment]
It's commented out in code, not CSS https://github.com/vladmandic/automatic/blob/e8d8dae4c71f3bc5043e972620c37921cdf7def3/modules/ui.py#L163

(Again not the biggest deal for me to make local code changes, it won't be the only one.  But was wondering why it was missing.)

**Comment by vladmandic** [comment]
ok, its back (commit to github later today)
